src/legacy/server/saved_objects/service/lib/scoped_client_provider.ts

```diff
@@ -17,30 +17,55 @@
  * under the License.
  */
 import { PriorityCollection } from './priority_collection';
+import { SavedObjectsClientContract } from '..';
+
+export interface SavedObjectsClientWrapperOptions<Request = any> {
+  client: SavedObjectsClientContract;
+  request: Request;
+}
+
+export type SavedObjectsClientWrapperFactory<Request = any> = (
```

Comment:
is there any benefit to use `any` instead of `unknown`? btw it's a default behaviour since TS v3.5

Reply:
Yeah I agree, unknown would be safer here.